api/typeDefs/wallet.js

```diff
@@ -124,6 +124,11 @@ const typeDefs = `
     itemAct: ItemAct
   }
 
+  type InvoiceForward {
+    invoice: Invoice
+  }
```

Comment:
If this invoice does not belong to the same user as the withdrawal, this reveals the sender.

For lnurlp and receives, these users are the same but for p2p zaps they are different. 

While the UI would not have exposed this privacy leak, the API does. e.g. If I receive a p2p zap, I can determine who zapped me by querying the API for my withdrawl and ask it to include the invoice that corresponds to it.